docs/configuration/configuration.md

```diff
@@ -47,53 +47,44 @@ The other placeholders are specified separately.
 
 A valid example file can be found [here](/config/testdata/conf.good.yml).
 
-The global configuration specifies parameters that are valid in all other configuration
+### `<global>`
+
+The `global` section contains the global Prometheus server
+configuration. The global configuration specifies parameters that are valid in all other configuration
 contexts. They also serve as defaults for other configuration sections.
 
 ```yaml
 global:
-  # How frequently to scrape targets by default.
-  [ scrape_interval: <duration> | default = 1m ]
+# How frequently to scrape targets by default.
+[ scrape_interval: <duration> | default = 1m ]
+
+# How long until a scrape request times out.
```

Comment:
This should mention this is a default

Reply:
I didn't update any of the descriptive text from the options - they are copied wholesale. I'll take a look.